### ISSUE TITLE
GVN will not login (2015)

### ISSUE THREAD

**Issue by Corruptinglyneedful** [davuxcom/GVNotifierWPF#11]
I can't get this to login

I think it needs some sort of update to fix it.

I click Login and it does nothing.  It says Logging in, Waiting, Redrecting, and goes right back to the Log in screen.

No error and all user info is correct.

I'm on Windows 10

It was working just fine, up until today.

**Comment by AJolly** [comment]
Yeah, it certainly needs an update - google's been changing some of their secondary login schemes for security reasons.  I'm not sure the author will get around to fixing it.  The simplest would be to have it open up an embedded web browser to finish the login process.

**Comment by Corruptinglyneedful** [comment]
@AJolly it actually started working after I emailed him so, he might have fixed it.  I don't know.  It is working right now for me.

If he's done updating, one of ya'll should take over.  I have to have this program or some alternative.

**Comment by JaredTurner** [comment]
I am currently having the same issue. Do you have any suggestions other than:

>  The simplest would be to have it open up an embedded web browser to finish the login process.

**Comment by Corruptinglyneedful** [comment]
Strange, I don't have this issue right now.

Is there some alternative?

**Comment by vnikolov** [comment]
Does not work for me, with app specific password I can see a brief flash of "Redirecting.." and then it's back to login screen. Too bad, I loved this little app.
